README.md

```diff
@@ -30,68 +30,50 @@ We have few paths to lower down to the Torch MLIR Dialect.
 
-## Examples
-There are few examples of lowering down via path from PyTorch to MLIR and using the “mlir-cpu-runner” to target a CPU backend. Obviously this is just a starting point and you can import this project into your larger MLIR projects to continue lowering to target GPUs and other Accelerators.
-
-## Project Communication
-
-- `#torch-mlir` channel on the LLVM [Discord](https://discord.gg/xS7Z362)
-- issues/PR's on this github repo
-- [`torch-mlir` section](https://llvm.discourse.group/c/projects-that-want-to-become-official-llvm-projects/torch-mlir/41) of LLVM Discourse
-
-## Repository Layout
-
-The project follows the conventions of typical MLIR-based projects:
-
-* `include/torch-mlir`, `lib` structure for C++ MLIR compiler dialects/passes.
-* `test` for holding test code.
-* `tools` for `torch-mlir-opt` and such.
-* `python` top level directory for Python code
-
-## Interactive Use
-
-The `build_tools/write_env_file.sh` script will output a `.env`
-file in the workspace folder with the correct PYTHONPATH set. This allows
-tools like VSCode to work by default for debugging. This file can also be
-manually `source`'d in a shell.
-
-## Build Instructions
+## Check out the code
 
 ```shell
-# From checkout directory.
-git submodule init
-git submodule update
-
-# Use clang and lld to build (optional but recommended).
-LLVM_VERSION=10
-export CC=clang-$LLVM_VERSION
-export CXX=clang++-$LLVM_VERSION
-export LDFLAGS=-fuse-ld=$(which ld.lld-$LLVM_VERSION)
-
-# Install PyTorch. We currently track and require the nighly build.
-# If a usable PyTorch package is installed, the default cmake settings will
-# enable the PyTorch frontend.
-pip3 install --pre torch torchvision -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
-
-# Invoke CMake and build. This will also run all unit tests.
-./build_tools/build_standalone.sh
+git clone https://github.com/llvm/torch-mlir
+cd torch-mlir
+git submodule update --init
+```
 
-# Run write_env_file.sh to emit a .env file with needed
-# PYTHONPATH setup.
-./build_tools/write_env_file.sh
-source .env
+## Setup your Python Environment
 
+```
+python3 -m venv mlir_venv
+source mlir_venv/bin/activate
+pip3 install --upgrade pip #Some older pip installs may not be able to handle the recent PyTorch deps
+# Install latest PyTorch nightlies
+pip3 install --pre torch torchvision pybind11 -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
 ```
 
+## Build 
+```
+cmake -GNinja -Bbuild \
+  -DCMAKE_C_COMPILER=clang-13 \
+  -DCMAKE_CXX_COMPILER=clang++-13 \
+  -DLLVM_ENABLE_PROJECTS=mlir \
+  -DLLVM_EXTERNAL_PROJECTS=torch-mlir \
+  -DLLVM_EXTERNAL_TORCH_MLIR_SOURCE_DIR=`pwd` \
+  -DMLIR_ENABLE_BINDINGS_PYTHON=ON \
+  -DLLVM_TARGETS_TO_BUILD=host \
+  external/llvm-project/llvm
+
+# Additional quality of life CMake flags:
+# Enable ccache:
+#   -DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_CXX_COMPILER_LAUNCHER=ccache
```

Comment:
Probably want lld in here. It reduces some of the link jobs from minutes to seconds, especially in debug builds.

Reply:
done